src/multi_auth/providers/google.cr

```diff
@@ -84,12 +84,18 @@ class MultiAuth::Provider::Google < MultiAuth::Provider
     @json = JSON.parse(raw_json)
     raise json["error"]["message"].as_s if json["error"]?
 
-    name = primary("names")
+    name = if primary?("names")
+      primary("names")
+    else
+      JSON::Any.new({} of String => JSON::Any)
+    end
+
+    display_name = name["displayName"]?.to_s
 
     user = User.new(
       "google",
       json["resourceName"].as_s,
-      name["displayName"].as_s,
+      display_name,
```

Comment:
This field wasn't required in the `User` anyway, so this can be an empty string. Maybe `nil` is better here?